package.json

```diff
@@ -52,23 +51,22 @@
     "build-url": "^6.0.1",
     "case": "^1.6.3",
     "clipboard": "^2.0.8",
+    "core-js": "^3.16.1",
     "deepmerge": "^4.2.2",
     "filesize": "^8.0.2",
     "focus-trap": "^5.1.0",
     "focus-trap-vue": "0.0.6",
+    "glob": "^7.1.6",
     "js-cookie": "^2.2.1",
     "lodash.clonedeep": "^4.5.0",
     "lodash.debounce": "^4.0.8",
     "lodash.findindex": "^4.6.0",
     "lodash.isempty": "^4.4.0",
     "lodash.sortby": "^4.7.0",
     "nuxt": "^2.15.4",
-    "nuxt-ssr-cache": "^1.5.2",
-    "patch-package": "^6.4.7",
     "reakit-utils": "^0.15.2",
-    "sass": "^1.34.0",
-    "sass-loader": "^10.1.1",
     "uuid": "^8.3.2",
+    "vue-i18n-extract": "^2.0.0",
```

Comment:
This is the package for extracting any untranslated strings from the app, and not needed in prod, so it should be a dev dependency.

Reply:
It is not needed for the `i18n:get-translations` script? Because I think the intention is to be able to run that command in production as well.

Reply:
Sorry I haven't checked for replies, @krysal . No, `vue-i18n-extract` shouldn't be used. I did copy some parts of it in `parse-vue-files` 😁

Reply:
Moved in [`1875f10`](https://github.com/WordPress/openverse-frontend/pull/415/commits/1875f10a18190cb005559f8296f5c061dbef2e76). Thanks!